src/Simulation/qdk_sim_rs/.gitignore

```diff
@@ -0,0 +1,12 @@
+linux
+osx
+win10
+target
+
+# We inject version numbers into Cargo.toml, so don't want them stored in repo.
```

Comment:
I'm curious why you chose this strategy. Is it possible to leave the Cargo.toml checked in and instead have it load version numbers from environment variables or something similar? I only ask because I worry that having Cargo.toml not checked in might set us up for cases where dev tools helpfully automatically update the local file and the change gets missed in commit and causes a failing PR build.

Reply:
Agreed entirely. I was unable to find a way to inject version numbers from the environment, but if you have any suggestions I would strongly prefer to do it that way for all the reasons you lay out in your comment.

Reply:
Ah, I wasn't sure if you had already tried something like that. It's certainly possible the template-style approach is our best bet then. I'll circle back if I happen to find another approach worth trying!

Reply:
Resolving for now, but happy to revisit this if you find anything. Would definitely love to have something more robust than templating!